### DISCUSSION TITLE
3.0.0-M2

### DISCUSSION THREAD

**Discussion by Him188** [open-ani/animeko#91] [Announcements]
3.0.0 第二个里程碑版本. 主要是重新设计了追番列表, 支持弹幕, 优化了视频缓冲速度.

- 初步支持桌面端
   - 目前只有手机 UI
   - 不支持视频播放功能
   - 可能没有经过全面测试
   - *至少能跑了*

### 我的追番

- 重新设计追番列表
   - 增加控件可交互面积
   - 仅展示"继续播放"的按钮, 将选集移入二级菜单
- 在点击看过时弹出对话框设置所有剧集为看过
- 修复番剧是否正在播出的判定 (UI 问题)
- 支持按想看/看过等状态分类 (#73)
- 在离开页面回到追番列表时, 恢复列表滚动位置
- 修复一些布局细节问题

### 视频

- 支持视频弹幕 (目前不支持关闭和设置样式)
- 优化视频下载速度, 降低首帧等待时间
   - *观看芙莉莲这种热门资源十秒内可以开播, 但最坏情况还是得等二三十秒.*
   *这个问题感觉只能未来做自动提前缓存来解决, 毕竟 bt 下载就是有速度问题*
- 视频字幕增加黑色边框
- 修复登录 Bangumi 相关的问题
- 调整播放视频时按返回键的行为
- 在播放视频时阻止屏幕息屏
- 在息屏或者放到后台时自动暂停播放, 当回到页面时自动恢复播放

----

**附件:**

- `.zip` 是包含 JRE 的解压即用的绿色版本；
- `.dmg`、`.deb`、`.rpm` 和 `.msi` 分别是 macOS、Debian、RedHat 和 Windows 平台的安装程序；
- `.exe` 是包含 JRE 的 Windows 程序，可以直接运行。
   - ***备注: Windows 平台打包出了点问题, 之后有空补上***
- `.jar` 是不包含 JRE 的最小程序，需要配合 JRE 11 及以上使用；
- `.apk` 是 Android 应用，支持的最低 Android 版本为 8.0；

注意，桌面平台仅支持 64 位操作系统。在使用 macOS 的安装程序时可能提示文件损坏，这是因为此版本应用没有签名，可以忽略错误继续使用。

也可扫描以下二维码下载 Android 版本：
![](https://github.com/Him188/ani/releases/download/v3.0.0-M2/ani-3.0.0-M2.apk.qrcode.png)

**Comment by NeKoOuO** [comment]
讚喔😋

**Comment by Him188** [comment]
刚刚看了一集, 感觉做播放器手势尤其是跳 op 迫在眉睫
